src/sys/ioctl/mod.rs

```diff
@@ -35,7 +35,7 @@
 //! from [rust-spidev](https://github.com/posborne/rust-spidev).
 //!
 //! ```
-//! #[macro_use] extern crate nix;
```

Comment:
You should be actually able to completely remove this line. This is automatically inserted by the doctest script.